pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -90,20 +85,17 @@ var _ = Describe("Pod Network", func() {
 	var mtu int
 	var cacheFactory cache.InterfaceCacheFactory
 	var libvirtUser string
-	var newPodNIC = func(vmi *v1.VirtualMachineInstance) podNIC {
-		return podNIC{
-			cacheFactory: cacheFactory,
-			handler:      mockNetwork,
-			vmi:          vmi,
+	var noLauncherPID *int
```

Comment:
The `no` prefix is confusing. At first glance I thought it is meaning there is no launcher pid.

Reply:
Created a new test helper constructor "newPodNICWithMocksWithoutLauncherPID" so we don't have to use a var and it's explicit.